src/vs/workbench/contrib/terminal/common/terminalConfiguration.ts

```diff
@@ -660,3 +666,20 @@ export function registerTerminalConfiguration() {
 	const configurationRegistry = Registry.as<IConfigurationRegistry>(Extensions.Configuration);
 	configurationRegistry.registerConfiguration(terminalConfiguration);
 }
+
+Registry.as<IConfigurationMigrationRegistry>(WorkbenchExtensions.ConfigurationMigration)
+	.registerConfigurationMigrations([{
+		key: TerminalSettingId.EnableBell,
+		migrateFn: (enableBell, accessor) => {
+			const configurationKeyValuePairs: ConfigurationKeyValuePairs = [];
+			configurationKeyValuePairs.push([TerminalSettingId.EnableBell, { value: undefined }]);
```

Comment:
This is added twice to the array?

Reply:
not sure what you mean

Reply:
It's the same content on line 681

Reply:
🤦🏼‍♀️  the rare case when viewing the diff in vscode vs on github makes things clearer.